src/astx/exceptions.py

```diff
@@ -46,3 +50,114 @@ def get_struct(self, simplified: bool = False) -> ReprStruct:
         key = "THROW-STMT"
         value = self.exception.get_struct(simplified) if self.exception else ""
         return self._prepare_struct(key, value, simplified)
+
+
+@public
+@typechecked
+class CatchHandlerStmt(StatementType):
+    """AST class for catch statements."""
+
+    body: ASTNodes[Expr]
```

Comment:
```suggestion
    body: ASTNodes[AST]
```

Thinking a bit more about that ... we cannot restrict it to just Expr ... it could be also Stmt as well ... for example `return` `yield` ... etc